pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+python_files=tests/*.py
+norecursedirs = __pycache__
```

Comment:
Does this module work with Python 3? If not we should maybe remove this line, lest people start making assumptions, then accusations and demands.